README.md

```diff
@@ -7,17 +7,37 @@ A list of awesome mobbing/ensemble/team programming resources. PRs are very welc
 ### Web
 
  - [mobti.me](https://mobti.me) - A free, collaborative, real-time mobbing timer.
+ - [mob.sh](https://mob.sh) - A free, script which gives you a simple set of shell commands for a fast handover.
+ - [timer.mob.sh](https://timer.mob.sh) - A timer, for the mob.sh script.
```

Comment:
Can you indent this and the following points, since they are part of the mob.sh ecosystem, please and thanks?

Reply:
makes sense